packages/crypto-wallet-core/package.json

```diff
@@ -27,12 +27,12 @@
   "author": "Micah Riggan",
   "license": "MIT",
   "dependencies": {
-    "@abcpros/bitcore-lib": "^8.25.31",
-    "@abcpros/bitcore-lib-cash": "^8.25.31",
-    "@abcpros/bitcore-lib-doge": "^8.25.31",
-    "@abcpros/bitcore-lib-ltc": "^8.25.31",
-    "@abcpros/bitcore-lib-xec": "^8.25.36",
-    "@abcpros/bitcore-lib-xpi": "^8.25.36",
+    "@abcpros/bitcore-lib": "^8.25.32",
```

Comment:
"version": "8.25.34",
This version is not bump? We'll need to bump the version and all of its dependencies